src/main/java/seedu/address/logic/parser/Option.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.parser;
+
+/**
+ * A prefix that marks the beginning of an argument in an arguments string.
+ * E.g. 't/' in 'add James t/ friend'.
+ */
+public class Option {
```

Comment:
We should make sure that parts of code that require option use this class for code standardisation.

Reply:
yup agree

Reply:
Ok, we can standardise to use the Option class to represent a option instead of using a string, similar to prefix.
i'll just tag everyone and myself to remind us to use it @edelinetenges @nicoleang09 @Stratostorm @mechastriker3 @clarlzx